cluster/aws/util.sh

```diff
@@ -737,6 +724,27 @@ function kube-down {
       done
     fi
 
+    echo "Deleting instances in VPC: ${vpc_id}"
+    instance_ids=$($AWS_CMD --output text describe-instances \
+                            --filters Name=vpc-id,Values=${vpc_id} \
+                                      Name=tag:KubernetesCluster,Values=${CLUSTER_ID} \
+                            --query Reservations[].Instances[].InstanceId)
+    if [[ -n ${instance_ids} ]]; then
+      $AWS_CMD terminate-instances --instance-ids $instance_ids > $LOG
+      echo "Waiting for instances to be deleted"
+      while true; do
+        instance_states=$($AWS_CMD describe-instances --instance-ids $instance_ids | expect_instance_states terminated)
+        if [[ "$instance_states" == "" ]]; then
+          echo "All instances deleted"
+          break
+        else
+          echo "Instances not yet deleted: $instance_states"
```

Comment:
nit: ${instance_states} (while we're at it)

(there's a ton of non-compliance with that one, but I'm only noting the new ones)